src/components/PostDetails.tsx

```diff
@@ -104,23 +65,23 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && (
+        {loading && (
           <Loader />
         )}
 
-        {loaded && hasError && (
+        {loading && error && (
```

Comment:
```suggestion
        {!loading && error && (
```
It shows an error during loading:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/03cf51ea-302f-4c72-9a4a-05d08de3a9ff)